src/main/scala/org/apache/mesos/chronos/scheduler/jobs/TaskUtils.scala

```diff
@@ -32,6 +32,15 @@ object TaskUtils {
     taskIdPattern.findFirstIn(taskIdString).nonEmpty
   }
 
```

Comment:
couldn't get it to work, probably because scalaz is not a direct dependency in the maven project, not sure. Will simply keep as if/else